.gitignore

```diff
@@ -32,3 +32,4 @@ package-lock.json
 /var/
 /vendor/
 ###< symfony/framework-bundle ###
+.idea
```

Comment:
Should not have been commited. You should ignore it through your global `.gitignore`.